src/service/resources-impl.js

```diff
@@ -1402,6 +1408,37 @@ export class Resources {
       // Selection API not supported.
     }
   }
+
+  /**
+   * Cleanup task queues from tasks for elements that has been unloaded.
+   * @param resource
+   * @private
+   */
+  maybeRemoveFromQueues_(resource) {
```

Comment:
So the maybe part is two parts:
- Has the resource been unlaid out (by the indication of NOT_LAID_OUT) - This is only changed when the unlayoutCallback actually returns true. 
- Is the task actually in the queue?

I wasn't sure what to call it. Maybe rename it to something like `removeStaledTasks`, `cleanupQueuesFromStaledTasks`?